### PR TITLE
Fix #1214: Standalone dev run

### LINKED ISSUE #1214: Standalone dev run
Make possible to run dev environment as standalone.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

**Review by romanstrobl — CHANGES_REQUESTED** [review]
We should include the Next Step and WebFlow client applications, too.

File: powerauth-webflow-client/pom.xml [hunk 1]
```diff
@@ -148,6 +148,15 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```
Comment (romanstrobl) [inline powerauth-webflow-client/pom.xml:151]: The client does not require connection to Postgres as far as I know.

**Review by romanstrobl — APPROVED** [review]
Approved, however I would suggest to remove database-related configuration for powerauth-webflow-client, it should not be required.

File: powerauth-webflow-client/pom.xml (hunk 1 above)
Comment (banterCZ) [reply powerauth-webflow-client/pom.xml:151]: Good catch, I have change that to H2.

**Pull request merged**